vector/src/main/java/im/vector/app/features/settings/labs/VectorSettingsLabsFragment.kt

```diff
@@ -140,4 +147,16 @@ class VectorSettingsLabsFragment :
     private fun onNewLayoutPreferenceClicked() {
         configureUnreadNotificationsAsTabPreference()
     }
+
+    private fun configureEnableClientInfoRecordingPreference() {
+        findPreference<VectorSwitchPreference>(VectorPreferences.SETTINGS_LABS_CLIENT_INFO_RECORDING_KEY)?.onPreferenceChangeListener =
+                OnPreferenceChangeListener { _, newValue ->
+                    when {
```

Comment:
Maybe change to `when (newValue as? Boolean)`

Reply:
Updated in [189e772](https://github.com/vector-im/element-android/pull/7352/commits/189e77211e14b4e1e1eca8ae0dcce4753fe92ae7)